src/colvarcomp_rotations.cpp

```diff
@@ -131,9 +127,13 @@ void colvar::orientation::apply_force(colvarvalue const &force)
   cvm::quaternion const &FQ = force.quaternion_value;
 
   if (!atoms->noforce) {
+    rotation_derivative<cvm::atom_pos, cvm::atom_pos> rot_deriv(rot, ref_pos, shifted_pos);
```

Comment:
Can we perhaps store a pointer in the classes that use a rotation explicitly, or possibly even in the `rotation` class itself? We can't forward declare a pointer to a templated class, but we can do this:
```
// Header
struct rotation_derivative_fwd;
class SomeClass {
  rotation_derivative_fwd *rot_deriv;
};

// Implementation CPP file
struct rotation_derivative_fwd :
  public rotation_derivative<cvm::atom_pos, cvm::atom_pos> {};
```

Reply:
Done